profiles/consul-snapshots.nix

```diff
@@ -0,0 +1,66 @@
+{ config, pkgs, ... }:
```

Comment:
where is this file coming from?

Reply:
cc @johnalotoski. Is this supposed to exists?

Reply:
Not there, this file was already merged as a module file, along with vault and nomad snapshot files in this [PR](https://github.com/input-output-hk/bitte/pull/68).